packages/keys/src/key-pair.ts

```diff
@@ -9,3 +11,15 @@ export async function generateKeyPair(): Promise<CryptoKeyPair> {
     );
     return keyPair as CryptoKeyPair;
 }
+
+export async function createKeyPairFromBytes(bytes: Uint8Array, extractable?: boolean): Promise<CryptoKeyPair> {
+    if (bytes.length !== 64) {
+        // TODO: Coded error.
+        throw new Error('KeyPair bytes must be of length 64');
+    }
+    const [publicKey, privateKey] = await Promise.all([
+        crypto.subtle.importKey('raw', bytes.slice(32), 'Ed25519', true, ['verify']),
```

Comment:
For the next poor person who has to read this, can you comment the inscrutable positional args…
```suggestion
        crypto.subtle.importKey(
            'raw',
            bytes.slice(32),
            'Ed25519',
            /* extractable */ true,
            ['verify'],
        ),
```